src/blog/_posts/2023-08-01-introduction-to-cadence-signals.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introduction to Cadence Signals
+
+date: 2023-08-06
```

Comment:
It would be good to move the date to be in September otherwise it wont come up at the top of the list when it is published.